src/components/policy/policy_regular/src/cache_manager.cc

```diff
@@ -103,6 +103,7 @@ CacheManager::CacheManager()
     : CacheManagerInterface()
     , pt_(new policy_table::Table)
     , backup_(new SQLPTRepresentation())
+    , settings_(nullptr)
```

Comment:
Reorder warning is occurring on Travis. `settings_` should be initialized after `update_required`.